packages/next/src/server/app-render/app-render.tsx

```diff
@@ -765,7 +849,7 @@ async function renderToHTMLOrFlightImpl(
           }}
         >
           <ServerInsertedHTMLProvider>
-            <ServerComponentsRenderer asNotFound={asNotFound} />
+            <ReactServerEntrypoint />
```

Comment:
no need to forward props through the wrapper, we just render the right props on our RSC App component straight away (See above)